Skills.md

```diff
@@ -1,61 +1,80 @@
-anything".
-
 For example:
 Robert wants to ask a villager for help. The GM asks how he wants to do so. Robert says that he wants to plea, pulling on the
 villagers heart strings. The GM doesn't see any physical impact from the action, and determines the Robert is trying to actively
-change the villagers mind, so he calls for an easy 'lww' (Lightning, Water, Water) test, as lightning represents change,
+change the villagers mind, so he calls for a 'Water, Lightning, Water, threshold 8' test, as lightning represents change,
 a water the 'aggressive interal' actions. Robert's character currently has the following grid:
 L: f(1), e(1)
 F: f(1)
 E: l(1), s(1), w(1)
 W: w(1)
 S: Nothing.
 
-So his threshold tests are (L:5(2 not lightning gems, +1 for easy) on d8, W:4(1 water gem, +1 for easy) on d6,
-W:4(1 water gem, +1 for easy) on d6).
+The GM has decided that the Major element is Water, and the Minor elements are Water and Lightning.
+
+Robert has 2 water gems, so from the table above, we can see he should roll a d10 for the Major test, and again for the Minor.
+He has 1 lightning gem so for the lightning test, he rolls only a d8.
+
+Robert rolls 6 for his Major water test, 7 for his Minor water test, and 4 for his lightning test.
+Robert now looks at the gems in his spheres. For water, he has only a single water gem, so he adds 2 to each of his
+water rolls, resulting in 8 for his Major, and 9 for his Minor.
+For lightning, he has 2 gems, but neither are lightning gems (fire and earth), so he adds 2 to his lightning test, 1 for each
+of the 2 gems, giving a result of 6.
+
+With this result, Major 8, and Minors of 9 and 6, he compares against the threshold of 8.
+His Major equals the threshold, and one of his minors exceeds it, so the test passes, and the villager is pursuaded.
+However, the lightning Minor failed, so the GM may decide that there is a side effect, associated with change, that
+was not desired. In this case, the GM decides that the Villager sees Robert's characters point, and does as requested,
+but feels that he might be being manipulated, and the GM may increase the threshold for Robert to change the Villagers
+mind again in future.
+
+Thresholds
+
+| Level           | Threshold | 50% pass    | >=75% pass  | >=90% pass  | 100% pass  |
+| --------------- | --------- | ----------- | ----------- | ----------- | ---------- |
+| Trivial         |  3        | None        | d8          | d10, 1n     | 2n         |
+| Easy            |  5        | d8          | d8, 1m      | d10, 1m, 1n | 5n         |
+| Average         |  7        | d8, 1m      | d8, 1m, 2n  | d10, 2m, 2n | 2m, 2n     |
+| Challenging     |  9        | d8, 1m, 2n  | d10, 2m     | d12, 3m, 1n | 3m, 2n     |
+| Hard            | 11        | d10, 2m, 1n | d12, 3m, 1n | None        | None       |
+| Very Hard       | 13        | d12, 3m     | None        | None        | None       |
+| Nigh Impossible | 15        | d12, 3m, 2n | None        | None        | None       |
```

Comment:
I think this was a good call. Raw thresholds are easier to parse than multiple modified results.

Reply:
Agreed.